be/btree_internal.h

```diff
@@ -34,13 +34,15 @@
 #include "format/format.h" /* m0_format_header */
 #include "be/btree.h"      /* BTREE_FAN_OUT */
 #include "be/btree_xc.h"   /* m0_be_btree_backlink_xc */
+#include "lib/types.h"
```

Comment:
add structure as comment for which this file is added

Reply:
Ok.
I will update it.